container-core/pom.xml

```diff
@@ -82,6 +82,31 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
This should probably be moved to container-search or similar. Container-core/disc should ideally only contain core jdisc functionality.

Reply:
I agree. It is due to com.yahoo.language.provider.DefaultLinguisticsProvider in container-core.